frontend/src/types.d.ts

```diff
@@ -1,5 +1,7 @@
 import { DeviceState, ServiceState } from 'remote.it'
 
+/// <reference types="@types/segment-analytics" />
```

Comment:
Don't use imports like this, instead import the type from the library like a regular import:

```js
import { SomeType } from 'segment'
```

You can see this in latest portal code.